### PR TITLE
New sigup dom

### PR DESCRIPTION
**Pull request by Anies12**

### CODE REVIEW

**Review by ahmedalmi95 — CHANGES_REQUESTED** [review]
white space

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 node_modules
-.env
-
+.env
```
Comment (Fatmasiam) [inline .gitignore]: add new line

File: public/js/signup.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    });
+            }
+        });
+    }
+});
+getElement(null, 'back2').addEventListener(() => {
+    fisrstBack.addEventListener('click', () => {
+        ToggleDisplay(thirdStepDiv, secondStepDiv);
+    });
+});
```
Comment (Fatmasiam) [inline public/js/signup.js]: and here add new line

File: src/views/signup.hbs [hunk 3]
```diff
@@ -1,4 +1,80 @@
...
+      <div class="password">
+         <label for="confirmPass">Confirm Your Password
+            <input id="confirmPass" class="confirmPass" type="password" name='confirmPass' id="confirmPass"
+               placeholder="Confirm Your Password" required>
+         </label>
+      </div>
+      <input type="submit" value="Back" id="back2">
+      <input type="submit" value="Sign Up" id="signUpBtn">
+   </div>
+</div>
```
Comment (Fatmasiam) [inline src/views/signup.hbs]: and here

**Comment by tomduggan85** [comment]
@Anies12 Hey 👋I added some comments to the other signup dom PR (#69) that apply here too.  I see now that #69 is closed and this one is probably the one you'll merge.

File: public/js/signup.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                fetch('/signup', {
+                    method: 'POST',
+                    credentials: 'same-origin',
+                    body: JSON.stringify(signupData),
+                    headers: { 'Content-Type': 'application/json' },
+                })
+                    .then(res => res.json())
+                    .then((res) => {
+                        if (res.error) {
+                            swal(text: `${res.error}`, type: "warning");
```
Comment (Fatmasiam) [inline public/js/signup.js]: how you give the swal key and value without object brackets ??!!

File: public/js/signup.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+                        } else {
+                            window.location.href = '/login';
+                        }
+                    })
+                    .catch(() => {
+                        swal(text: 'Error, please try again..', type: "warning");
```
Comment (Fatmasiam) [inline public/js/signup.js]: and here ?

**Pull request closed without merging**